ilastik/applets/objectClassification/objectClassificationDataExportGui.py

```diff
@@ -151,3 +156,16 @@ def _initPredictionLayers(self, predictionSlot):
                 layers.append(predictLayer)
 
         return layers
+
+    def _initColortableLayer(self, labelSlot):
+        objectssrc = createDataSource(labelSlot)
+        objectssrc.setObjectName("LabelImage LazyflowSrc")
+        ct = colortables.create_default_16bit()
+        ct[0] = QColor(0, 0, 0, 0).rgba()  # make 0 transparent
+        layer = ColortableLayer(objectssrc, ct)
+        layer.name = "Object Identities - Preview"
+        layer.setToolTip("Segmented objects, shown in different colors")
+        layer.visible = False
+        layer.opacity = 0.5
+        layer.colortableIsRandom = True
+        return [layer]
```

Comment:
I really wish all these layer parameters could be set inside `ColortableLayer.__init__`. Maybe we can get some patches to volumina to do that later

In fact, this looks eerily similar to the code in `objectExtractionGui.py `. If updating `ColortableLayer` in `volumina` would be too much of a hassle, than maybe this should be a `@(static|class)method` so that we can reuse it and don't replicate this logic around.

Reply:
totally agree - since this patch is targeted to finalize 1.3.3, I will open an Issue about this in volumina but will proceed with the changess for now